Test/civl/samples/snapshot.bpl

```diff
@@ -0,0 +1,116 @@
+type Value;
```

Comment:
If you look at other Boogie files in the samples folder, you will see two lines at the top
```
// RUN: %parallel-boogie "%s" > "%t"
// RUN: %diff "%s.expect" "%t"
```

These lines indicate that this file is a test. Please add these lines to this file.

A test x.bpl also has a "golden" file associated with it called x.bpl.expect. Please create this file also. 

Boogie tests are run using the lit tool which you have to install on your machine. See instructions on the Boogie github site.

An automatic way to create the golden file for a test is to run lit (after adding the two lines at the top which are lit directives). The output generated is dumped into the Output folder. You can grab it from there.